Week-7/Assignment7.5.js

```diff
@@ -0,0 +1,24 @@
+
+	function printNGE(arr, n)
+	{
+		var next, i, j;
+		for (i = 0; i < n; i++)
+		{
+		next = -1;
+		for (j = i + 1; j < n; j++)
+		{
+			if (arr[i] < arr[j])
+			{
+			next = arr[j];
```

Comment:
what is the time complexity here?

Reply:
its o(n2) can u optimize the code to o(n)

Reply:
any suggestion on how can I do this?

Reply:
u can make use of stacks

Reply:
try to think how u can solve using stacks 
if u r not getting then refer soln in geeks for geeks: https://www.geeksforgeeks.org/next-greater-element/#:~:text=Given%20an%20array%2C%20print%20the,next%20greater%20element%20as%20%2D1.